### ISSUE TITLE
Canal on k8s 1.16 logs excessively on Info level

### ISSUE THREAD

**Issue by superseb** [rancher/rancher#23559]
**What kind of request is this (question/bug/enhancement/feature request):**
bug

**Steps to reproduce (least amount of steps as possible):**
Launch k8s 1.16 custom cluster with Canal

**Result:**
Excessive logs on info level

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): v2.3.1
- Installation option (single install/HA): single

**Comment by jiaqiluo** [comment]
The bug is reproduced in a cluster with k8s 1.16.2 in Rancher:v2.3.1

There are excessive logs in the logs if the container `calico-node` 

```
2019-10-21 21:34:21.845 [INFO][33] ipsets.go 223: Asked to resync with the dataplane on next update. family="inet"
2019-10-21 21:34:21.845 [INFO][33] ipsets.go 306: Resyncing ipsets with dataplane. family="inet"
2019-10-21 21:34:21.847 [INFO][33] ipsets.go 356: Finished resync family="inet" numInconsistenciesFound=0 resyncDuration=1.351613ms
2019-10-21 21:34:21.847 [INFO][33] int_dataplane.go 959: Finished applying updates to dataplane. msecToApply=1.5475160000000001
2019-10-21 21:34:22.444 [INFO][33] int_dataplane.go 945: Applying dataplane updates
2019-10-21 21:34:22.444 [INFO][33] table.go 793: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="filter"
2019-10-21 21:34:22.446 [INFO][33] table.go 513: Loading current iptables state and checking it is correct. ipVersion=0x4 table="filter"
2019-10-21 21:34:22.449 [INFO][33] int_dataplane.go 959: Finished applying updates to dataplane. msecToApply=4.679328
2019-10-21 21:34:32.082 [INFO][33] int_dataplane.go 945: Applying dataplane updates
2019-10-21 21:34:32.082 [INFO][33] ipsets.go 223: Asked to resync with the dataplane on next update. family="inet"
2019-10-21 21:34:32.082 [INFO][33] ipsets.go 306: Resyncing ipsets with dataplane. family="inet"
2019-10-21 21:34:32.083 [INFO][33] ipsets.go 356: Finished resync family="inet" numInconsistenciesFound=0 resyncDuration=1.32073ms
2019-10-21 21:34:32.083 [INFO][33] int_dataplane.go 959: Finished applying updates to dataplane. msecToApply=1.489176
2019-10-21 21:34:42.271 [INFO][33] int_dataplane.go 945: Applying dataplane updates
2019-10-21 21:34:42.271 [INFO][33] ipsets.go 223: Asked to resync with the dataplane on next update. family="inet"
2019-10-21 21:34:42.271 [INFO][33] ipsets.go 306: Resyncing ipsets with dataplane. family="inet"
2019-10-21 21:34:42.273 [INFO][33] ipsets.go 356: Finished resync family="inet" numInconsistenciesFound=0 resyncDuration=1.593686ms
2019-10-21 21:34:42.273 [INFO][33] int_dataplane.go 959: Finished applying updates to dataplane. msecToApply=1.9337309999999999
2019-10-21 21:34:52.452 [INFO][33] int_dataplane.go 945: Applying dataplane updates
2019-10-21 21:34:52.452 [INFO][33] ipsets.go 223: Asked to resync with the dataplane on next update. family="inet"
2019-10-21 21:34:52.453 [INFO][33] ipsets.go 306: Resyncing ipsets with dataplane. family="inet"
2019-10-21 21:34:52.454 [INFO][33] ipsets.go 356: Finished resync family="inet" numInconsistenciesFound=0 resyncDuration=1.608932ms
2019-10-21 21:34:52.454 [INFO][33] int_dataplane.go 959: Finished applying updates to dataplane. msecToApply=1.823835
2019-10-21 21:35:02.824 [INFO][33] int_dataplane.go 945: Applying dataplane updates
2019-10-21 21:35:02.824 [INFO][33] ipsets.go 223: Asked to resync with the dataplane on next update. family="inet"
2019-10-21 21:35:02.824 [INFO][33] ipsets.go 306: Resyncing ipsets with dataplane. family="inet"
2019-10-21 21:35:02.825 [INFO][33] ipsets.go 356: Finished resync family="inet" numInconsistenciesFound=0 resyncDuration=1.290757ms
2019-10-21 21:35:02.825 [INFO][33] int_dataplane.go 959: Finished applying updates to dataplane. msecToApply=1.478046
```

**Comment by jiaqiluo** [comment]
The bug fix is validated in Rancher:v2.3-head 8d57a1cfb with the metadata from `dev` branch 

The above-mentioned info[33] lines do not show in the logs anymore
